cli/flox/Cargo.toml

```diff
@@ -8,6 +8,7 @@ edition = "2021"
 [dependencies]
 anyhow.workspace = true
 bpaf.workspace = true
+catalog-api-v1.workspace = true
```

Comment:
I feel pulling the client in in the cli crate is bad practice considering that we build a shim to avoid precisely that.

`type` declarations are actual aliases (unlike newtypes), we can at least reexport `api_types::ErrorResponse` from the shim which would allow us to drop the dependency on the api crate

Reply:
Added an alias. In this case the `flox` crate really is dependent on details of the API - it's looking at status code. Not sure how we're going to handle more specific errors but I'm wondering if we'll need to bubble context all the way up